### PR TITLE
Feature/setup deploy

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,47 @@
...
+{
+    "Version": "2008-10-17",
+    "Statement": [
+        {
+            "Sid": "PublicReadForGetBucketObjects",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "*"
+            },
+            "Action": "s3:GetObject",
```
Comment (sharph) [inline README.md]: This looks like a read only policy and will not allow s3_website to write to the bucket.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,47 @@
...
+Once the AWS access keys are set in your environment, deploy the app:
+```
+s3_website push --force
+```
+
+If you want to change the bucket the app deploys to, edit the `s3_bucket` setting in s3_website.yml
+
+If you change the bucket, ensure that you do the following:
+  - Add the bucket to the IAM S3-Full-Access policy for the user's access keys
+  - Configure the bucket for static hosting, with both the index and error pages pointing to `index.html`
```
Comment (sharph) [inline README.md]: `s3_website cfg apply` is the somewhat secret command that will apply all the settings necessary to the bucket.

File: README.md (hunk 1 above)
Comment (sharph) [inline README.md]: Ah, nevermind. I see what you are doing here. A policy statement for the IAM role for write access would be useful. And the command I've commented above should apply read access.

Comment (CloudNiner) [inline README.md]: Seems to work as expected. Updated README to use `s3_website cfg apply`

Comment (sharph) [inline README.md]: You should be able to drop the instructions about configuring the bucket for static hosting and the bucket policy.

Comment (CloudNiner) [inline README.md]: Sure, I left it there in case of issues, but w/e. Updated.

**Comment by sharph** [comment]
+1

**Pull request merged**